AgentDiscoveries-Frontend/app/src/components/login.js

```diff
@@ -103,10 +100,15 @@ export default class Login extends React.Component {
         });
     }
 
-    updateIsLoggedIn() {
+    isUserLoggedIn(){
         let token = window.localStorage.getItem("Token");
-        var isLoggedIn = token && true;
+        return token && true;

```

Comment:
Not seen the `token && true` pattern before - I'd usually do `!!token`.

Also that let can be const?